client/src/i18n/en/table.json

```diff
@@ -23,7 +23,7 @@
             "CANCEL_CASH": "Cancel Cash Payment",
             "CHARGE": "Charge",
             "CLIENT": "Client",
-            "CODE": "Code",
+            "CODE": "Registration number",
```

Comment:
It looks like this is used in other modules (like [stock](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/client/src/modules/stock/entry/entry.js#L72)) for the word "Code".  Could you make a new key for this translation?